.github/workflows/repolinter.yml

```diff
@@ -27,4 +28,4 @@ jobs:
         if: steps.check_files.outputs.files_exists == 'false'
         uses: todogroup/repolinter-action@v1
         with:
-          config_url: "https://raw.githubusercontent.com/quic/.github/main/repolint.json"
```

Comment:
nitpick.. but unrelated change here.

Reply:
that was surprisingly hard to fix. my editors fought me. had to fix it Python.